src/libsync/owncloudpropagator.h

```diff
@@ -382,6 +382,8 @@ private slots:
 private:
 
     bool scheduleDelayedJobs();
+
+    SyncFileItem::Status _hasError = SyncFileItem::Status::NoStatus;
```

Comment:
@mgallien I think the naming is misleading. I would expect it to be a bool member while it is an enum type. Could you name it something like `_errorStatus`?